### DISCUSSION TITLE
Unable to have a proof of enrolment in one photo

### DISCUSSION THREAD

**Discussion by Arison7** [community/community#51398] [GitHub Education]
Hello everyone,

I have quite a small issue but since my ticket was left unanswered I am asking here. It's a very specific problem regarding Polish proof of enrolment. 

As a polish student, my Student ID is two-sided and both sides are necessary to provide the validation needed. Unfortunately,  on the Educational pack application website, only one photo can be added.  The last time I applied the photo of me holding my ID in the mirror was accepted this time that didn't happen.  Does anyone know how to possibly contact GitHub in order to have the validation done via email, not the website? Or is there maybe another polish student out there who managed to overcome the same issue ( for the record the document I am referring to is "Legitimycja Szkolna"? 

Best regards,
Arison

**Comment by F0XN0T** [comment]
I would also recommend checking if your school has any guidance on how to provide proof of enrollment to third-party platforms like GitHub. They may have specific instructions or forms that could help you provide the necessary validation.

If you're still having trouble with the validation process, you could also try contacting GitHub's support team through their social media accounts or community forums. They may be able to provide additional guidance or escalate your case to a higher level of support.

I understand that this situation can be frustrating and time-consuming, but don't give up. Keep exploring different options and resources until you find a solution that works for you. Best of luck!
